code/core/src/common/js-package-manager/NPMProxy.ts

```diff
@@ -132,28 +132,28 @@ export class NPMProxy extends JsPackageManager {
     });
   }
 
-  public async findInstallations(pattern: string[]) {
```

Comment:
The idea of adding a depth option is because we want to try and get information over dependencies that are coming directly from package.json, but NOT as dependencies of dependencies. This needs further testing on all package managers